solana/modules/ntt-messages/src/ntt.rs

```diff
@@ -85,6 +102,53 @@ impl Writeable for NativeTokenTransfer {
         to.write(writer)?;
         to_chain.write(writer)?;
 
+        if A::SIZE != Some(0) {
+            let len: u16 = u16::try_from(additional_payload.written_size()).expect("u16 overflow");
+            len.write(writer)?;
+            // TODO: same as above
```

Comment:
This seems like a TODO to be fixed in a future PR. Should there be a longer sentence here indicating what needs to be done?

Reply:
This refers to a more verbose todo in `transceiver.rs` (and was copied from `ntt.rs`). Updating.